test/e2e/networks/ci.toml

```diff
@@ -61,7 +61,6 @@ database = "rocksdb"
 perturb = ["pause"]
 
 [node.validator05]
-block_sync = "v0" 
```

Comment:
should that be renamed to `block_sync_version`, rather than deleted? (I saw you've kept the version as part of the generation)

Reply:
That's right. Before, this line was needed to enable BlockSync, not just to asign the version.